src/components/Register.js

```diff
@@ -76,53 +53,41 @@ const Register = ({background}) => {
-
-                            <label> Enter a username </label>
-                            {invalidUsername ? <label className="invalid">Error: Username is already in use with another account. Try a different username!</label> : <label></label>}
-                            <input className="register-form-text-input" type="text" placeholder="Username" value={username} onChange={((e) => setUsername(e.target.value))}></input>
-
-
-                            <label> Enter a password </label>
-                            <input className="register-form-text-input" type="text" placeholder="Password" value={password} onChange={((e) => setPassword(e.target.value))}></input>
-
-
-                            <input className="register-form-submit-button" type="submit" placeholder="Login"></input>
-                        </div>
-                    </form>
-
-                
-                </div> }
-
-
-                <div className="">
-                    {LaptopScreen ?  <div></div> : <Footer />} 
-                </div>
-        
+    <div className={background}>
+        <div className="register-container">
+            <div className="back-arrow">
+                <Link to="/" style={{textDecoration: 'none'}}>
+                    <a href="#"><i class='bx bxs-left-arrow-circle'></i></a>
+                </Link>
+            </div>
+            {accountCreated ? <h2>Your account was created!</h2> : <h2>Register an account here!</h2>}
+            {accountCreated ? <div className="successful-register"><label>Click here to sign in!</label> <Link to="/" ><button className="register-form-submit-button"> Sign in!</button></Link></div>
+            : <div>
+                <form className="" onSubmit={onSubmit}>
+                    <div className="register-form">
+                        <label> Enter an email address </label>
+                        <input className="register-form-text-input" type="text" placeholder="Email" value={email} onChange={((e) => setEmail(e.target.value))}></input>
+                        <label> Enter a username </label>
+                        {invalidUsername ? <label className="invalid">Error: Username is already in use with another account. Try a different username!</label> : <label></label>}
+                        <input className="register-form-text-input" type="text" placeholder="Username" value={username} onChange={((e) => setUsername(e.target.value))}></input>
+                        <label> Enter a password </label>
+                        <input className="register-form-text-input" type="password" placeholder="Password" value={password} onChange={((e) => setPassword(e.target.value))}></input> 
+                                <ReCAPTCHA
+                                    sitekey="6LcgeDcdAAAAAOOgGLtrOpC5CcxujIZMLtyd-h33"
+                                    onChange={() => setIsVerified(false)} 
+                                />

```

Comment:
I think there should be some input verification messages for the user if they do not have the captcha clicked, for example "You must verify you are a human, click the box!" if the user tries to create the account without verifying. Similar implementation for missing username or password.